src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -82,8 +98,17 @@ public class InventoryWardBrowser extends ModalJFrame implements InventoryListen
             MessageBundle.getMessage("angal.common.user.col").toUpperCase()
     };
     private int[] pColumwidth = { 150, 150, 100, 100, 150 };
-    private JComboBox<Object> stateComboBox;
-    private JLabel stateLabel;
+    private JComboBox<String> statusComboBox;
+    private JLabel statusLabel;
+    JButton next;
+    JButton previous;
+    JComboBox pagesCombo = new JComboBox();
+    JLabel under = new JLabel("/ 0 Page");
```

Comment:
should this be localized?

Reply:
It's used on lines 258, 595, and 597 in different methods

Reply:
Ah ok! I will create a traduction key